sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuildInstaller.java

```diff
@@ -133,7 +165,11 @@ private static File locateInstalledScript(Version scannerVersion, File toDir, bo
     return script;
   }
 
-  private static String directoryName(Version scannerVersion) {
+  private static String directoryName(@Nullable Version scannerVersion) {
+    if(scannerVersion != null) {
     return "sonar-scanner-msbuild-" + scannerVersion.toString();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)